Framework/DataObjects/src/AffineMatrixParameter.cpp

```diff
@@ -128,6 +128,12 @@ AffineMatrixParameter &AffineMatrixParameter::operator=(const AffineMatrixParame
   if (this != &other) {
     this->m_affineMatrix = other.m_affineMatrix;
     this->m_isValid = other.m_isValid;
+    size_t nx = m_affineMatrix.numRows();
```

Comment:
Just noting down what was discussed in person. We shouldn't need to reallocate here as the lhs/rhs are the same size. We can leave the cppcheck warning as a false positive in this case.